src/options.ts

```diff
@@ -39,7 +57,16 @@ const options: BeautifierOptions = {
   },
   CSS: {
     indent_size: true,
-    indent_char: true,
+    indent_char: [
```

Comment:
These common options should be moved into an object variable and spread over their language specific usages here.